phpstan.neon.dist

```diff
@@ -2,3 +2,5 @@ parameters:
     level: 4
     paths:
         - lib
+    excludes_analyse:
+        - lib/Github/HttpClient/Plugin/HistoryTrait.php
```

Comment:
Can you add a comment at this line with the link to this issue? https://github.com/phpstan/phpstan/issues/3460

Hopefully this false positive can be fixed and this way we have some context on why this ignore is set!